lib/aida/bot.ex

```diff
@@ -1,19 +1,32 @@
 defmodule Aida.Bot do
-  alias Aida.{FrontDesk, Variable, Message, Skill, Logger, Channel, DataTable, Crypto}
+  alias Aida.{
+    Channel,
+    ChannelProvider,
+    Crypto,
+    DataTable,
+    DB.MessageLog,
+    DB.MessagesPerDay,
+    FrontDesk,
+    Logger,
+    Message,
+    Skill,
+    Variable
+  }
+
   alias __MODULE__
 
   @type message :: map
 
   @type t :: %__MODULE__{
-    id: String.t,
-    languages: [String.t],
-    front_desk: FrontDesk.t,
-    skills: [Skill.t],
-    variables: [Variable.t],
-    channels: [Channel.t],
-    public_keys: [binary],
-    data_tables: [DataTable.t]
-  }
+          id: String.t(),
```

Comment:
Why adding this syntax? Everywhere else we are avoiding unnecessary parentheses (especially for the dialyzer type definitions)